tests/test_drsclient.py

```diff
@@ -0,0 +1,281 @@
+        bundles=[did],
+        name=name,
+        guid=bundle_id,
+        size=size,
+        description=description,
+        version=version,
+        aliases=aliases,
+    )
+
+    # get with /bundle endpoint
+    rec1 = res1.json()
+    assert res1.status_code == 200
+    bundle_id = rec1["bundle_id"]
+    res2 = drs_client.get(bundle_id, "bundle", expand=True)
+    assert res2.status_code == 200
+    rec2 = res2.json()
+    assert rec2["id"] == bundle_id
+    assert rec2["name"] == rec1["name"]
+    assert rec2["size"] == size
+    assert rec2["description"] == description
+    assert rec2["version"] == version
+    assert rec2["aliases"] == aliases
+
+    # get with /ga4gh/drs/v1/objects endpoint
+    rec3 = drs_client.get(bundle_id, expand=True)
+    assert rec3.status_code == 200
+    rec3 = rec3.json()
+    assert rec3["id"] == bundle_id
+    assert rec3["name"] == rec1["name"]
+    assert rec3["size"] == size
+    assert rec3["description"] == description
+    assert rec3["version"] == version
+    assert rec3["aliases"] == aliases
+
+
+def test_async_get_bundle(index_client, drs_client):
+    rec = create_index_record(index_client)
+    did = rec["did"]
+    res1 = asyncio.run(drs_client.async_create(bundles=[did]))
+    rec1 = res1.json()
+    assert res1.status_code == 200
+    bundle_id = rec1["bundle_id"]
+
+    # get with /bundle endpoint
+    res2 = asyncio.run(drs_client.async_get(bundle_id, endpoint="bundle", expand=True))
+    assert res2.status_code == 200
+    rec2 = res2.json()
+    assert rec2["id"] == bundle_id
+    assert rec2["name"] == rec1["name"]
+
+    # get with /ga4gh/drs/v1/objects endpoint
+    res2 = asyncio.run(drs_client.async_get(bundle_id, expand=True))
+    assert res2.status_code == 200
+    rec2 = res2.json()
+    assert rec2["id"] == bundle_id
+    assert rec2["name"] == rec1["name"]
+
+
+def test_get_all(index_client, drs_client):
+
+    # create 100 files and bundles
+    for _ in range(100):
+        rec = create_index_record(index_client)
+        drs_client.create(bundles=[rec["did"]])
+
+    res1 = drs_client.get_all(limit=300)
+    assert res1.status_code == 200
+    rec1 = res1.json()
+    assert len(rec1["drs_objects"]) == 200
+
+    res2 = drs_client.get_all(form="bundle", limit=300)
+    assert res2.status_code == 200
+    rec2 = res2.json()
+    assert len(rec2["drs_objects"]) == 100
+
+    res3 = drs_client.get_all(form="object", limit=300)
+    assert res3.status_code == 200
+    rec3 = res3.json()
+    assert len(rec3["drs_objects"]) == 100
+
+
+def test_async_get_all(index_client, drs_client):
+
+    # create 100 files and bundles
+    for _ in range(100):
+        rec = create_index_record(index_client)
+        drs_client.create(bundles=[rec["did"]])
+
+    res1 = asyncio.run(drs_client.async_get_all(limit=300))
+    assert res1.status_code == 200
+    rec1 = res1.json()
+    assert len(rec1["drs_objects"]) == 200
+
+
+def test_layered_bundle(index_client, drs_client):
```

Comment:
i feel like this might be testing indexd/indexclient, not drsclient. if this doesn't add coverage for this repo, we can remove it